packages/amplify-graphql-types-generator/src/cli.js

```diff
@@ -118,7 +119,7 @@ yargs
         addS3Wrapper: argv["add-s3-wrapper"]
```

Comment:
either change this line to or add a new line as: `complexObjectSupport: argv["complex-object-support"]`
You don't have a `complexObjectSupport` in the options object, which is referenced in the generate method if the target is swift.

Reply:
Good catch. Fixed